.circleci/config.yml

```diff
@@ -2,154 +2,16 @@ version: 2.1
 orbs:
   architect: giantswarm/architect@0.2.0
 
-e2eTest: &e2eTest
```

Comment:
Drop whole e2e test from the same repo

Reply:
Yes, the managedservices test only works with appr. We will add a new e2e test called basicapp that is compatible with App Catalog. We can re-enable this then.